.azure-pipelines/Windows-CI.yml

```diff
@@ -56,6 +56,13 @@ jobs:
   - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
     displayName: Add conda to PATH
 
+  # cmake 3.28.3 failed to find protobuf installed from conda-forge
```

Comment:
Weird. I wonder why.

Reply:
I wonder if something is missing in the cmake files.

Reply:
hope it is resolved in the next cmake version: https://gitlab.kitware.com/cmake/cmake/-/issues/25704